wscript

```diff
@@ -77,14 +81,18 @@ def configure(ctx):
     ctx.load('compiler_cxx')
 
     # Check required headers
-    req_headers = ['stdlib.h', 'string.h', 'unistd.h', 'stdint.h', 'stdio.h', 'jpeglib.h']
+    req_headers = ['stdlib.h', 'string.h', 'stdint.h', 'stdio.h']
+    if platform.system() != "Windows":
```

Comment:
As mentioned in a previous comment, I am hesitant to use platform.system(). An alternative here would be to check that a build either has all linux flavors or all win32 and use that as a hint about linux vs windows. So, this should in theory allow us to cross-compile on linux for windows with mingw32 (never tried it myself though, so I may be missing something).

Reply:
Just want to note that this is not a blocker. We can start with platform.system(), and move to something else in the future if there is ever an interest for linux<->windows cross compilation.

Reply:
Done.